### PR TITLE
Make MaxVUs equal to preallocatedVUs when missing for arrival rate executors

### LINKED ISSUE #1317: maxVUs should be optional for arrival-rate executors
By default, `maxVUs` should probably just be equal to `preAllocatedVUs`, so the configuration of arrival-rate executors is slightly easier.

### PR DESCRIPTION
**Pull request by MStoykov**
fixes #1317

### CODE REVIEW

File: lib/executor/constant_arrival_rate.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func (carc ConstantArrivalRateConfig) Validate() []error {
 	}
 
 	if !carc.MaxVUs.Valid {
-		errors = append(errors, fmt.Errorf("the number of maxVUs isn't specified"))
+		carc.MaxVUs.Int64 = carc.PreAllocatedVUs.Int64
```
Comment (na--) [inline lib/executor/constant_arrival_rate.go:138]: Hmm this seems like a bit of a hack to me... I'd expect `Validate()` to not make any modifications, but yeah, unfortunately there's no other way to do it currently, without significant refactoring... :disappointed: That refactoring is probably going to have to be done as a part of https://github.com/loadimpact/k6/issues/1499 / https://github.com/loadimpact/k6/issues/1427. Until then, can you please add some code comments here, and also add a not in https://github.com/loadimpact/k6/issues/1427 to fix this code as well?

File: lib/executor/ramping_arrival_rate.go [hunk 2]
```diff
@@ -124,7 +124,7 @@ func (varc RampingArrivalRateConfig) Validate() []error {
 	}
 
 	if !varc.MaxVUs.Valid {
-		errors = append(errors, fmt.Errorf("the number of maxVUs isn't specified"))
+		varc.MaxVUs.Int64 = varc.PreAllocatedVUs.Int64
```
Comment (na--) [inline lib/executor/ramping_arrival_rate.go:127]: Same here

File: lib/executor/constant_arrival_rate.go (hunk 1 above)
Comment (MStoykov) [reply lib/executor/constant_arrival_rate.go:139]: fixed in  43ab37d

File: lib/executor/ramping_arrival_rate.go (hunk 2 above)
Comment (MStoykov) [reply lib/executor/ramping_arrival_rate.go:128]: fixed in  43ab37d

File: lib/executor/constant_arrival_rate.go (hunk 1 above)
Comment (na--) [reply lib/executor/constant_arrival_rate.go:139]: leaving this unresolved, so it's even more apparent that it needs to be fixed

**Comment by MStoykov** [comment]
The problem is that the refactor will need to literally touch every executor and most of the code that does anything with them, which is probably going to be:
1. too big to review quickly 
2. will break every other PR that does anything with the executors, I am for doing it after #1302 as that will probably make things somewhat easier :D (also is IMO more importatant for v0.27.0 release)

**Pull request merged**